packages/core/less/message-dialog.less

```diff
@@ -0,0 +1,26 @@
+.message-dialog-pf-body {
+  display: flex;
+  color: #363636;
+}
+
+.message-dialog-pf-icon {
+  font-size: 30px;
+  margin-right: 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.message-dialog-pf-text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
All of these can be removed, too. The children in this div can just be block-level elements.